### PR TITLE
WIP!!! (Not merge) - LPS-102097 - Needed changes to remove compatibility with BS3 layout

### PR DESCRIPTION
**Pull request by marcoscv-work**
Testing, WIP!

### CODE REVIEW

**Comment by marcoscv-work** [comment]
@john-co I need your help!! I've been doing smoke testing and it looks good but the commits:
https://github.com/liferay-frontend/liferay-portal/pull/497/commits/0cfc4e03a689bf4173578daadaea60608e5fe4ac
https://github.com/liferay-frontend/liferay-portal/pull/497/commits/fb4b7937edf40ea9fff46a135a23bcd415047313
https://github.com/liferay-frontend/liferay-portal/pull/497/commits/772e09800e6ec73417d51ea187200d5a7fe1b448

Are complex to test, some idea to help me with this? :-P

And the last commit:
https://github.com/liferay-frontend/liferay-portal/pull/497/commits/871a12b144c4cbc9ba8b52616b8f192d0e4c4bf2

I'm not sure why do we have those classes in source formatter, probably I shouldn't touch there 

Thanks!

**Comment by marcoscv-work** [comment]
About the Commerce modules changes, should we send them a PR separately?

https://github.com/liferay-frontend/liferay-portal/pull/497/commits/ccbd3d764b3f973aa0b6044bed8f9f161d7df851
https://github.com/liferay-frontend/liferay-portal/pull/497/commits/892fc95bef433ba69eccef141b25a73ef1132f3e
https://github.com/liferay-frontend/liferay-portal/pull/497/commits/0cfc4e03a689bf4173578daadaea60608e5fe4ac

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
ci:test:relevant

**Comment by marcoscv-work** [comment]
On the other hand, many modules are candidates to use `<clay:content*` taglibs, I would like to fix this task before, and then probably look for a way to planning migrate them to taglibs

**Comment by john-co** [comment]
We'll probably need some time to look more in detail to evaluate test coverage...all the QA's are occupied with the other Test Scenarios right now. But I agree that it would be good to involve the commerce team to review and test their related commits.

My initial thoughts are to evaluate how stable ci:test:relevant are. And if we want to run more tests if we're worried about risk, can run `ci:test` to run even more comprehensive ci testing.

**Comment by john-co** [comment]
ci:test:bundle

**Comment by john-co** [comment]
ci:test:poshi

**Comment by john-co** [comment]
Translations#TranslateWebContent issue looks flaky based on testray history: https://testray.liferay.com/home/-/testray/cases/1310779424

The modules-integration test is a known flaky issue. https://issues.liferay.com/browse/LPS-123052

**Comment by marcoscv-work** [comment]
> And some exploratory manual testing with various browser resizing across portal is probably good
> 

I've been trying to do this across portal and I think it looks correct but of course, 4 eyes see more than 2... and 6, more than 4 (I'm a genius)

**Comment by marcoscv-work** [comment]
I've just deleted the last commit

File: modules/apps/commerce/commerce-product-type-grouped-web/src/main/resources/META-INF/resources/render/view.jsp [hunk 1]
```diff
@@ -35,7 +35,7 @@ String productContentAuthToken = AuthTokenUtil.getToken(request, plid, CPPortlet
 		<div class="product-detail-header">
 			<div class="col-lg-6 col-md-7">
 				<div class="row">
-					<div class="col-lg-2 col-md-3 col-xs-2">
+					<div class="col-lg-2 col-md-3 col-2">
```
Comment (jbalsas) [inline modules/apps/commerce/commerce-product-type-grouped-web/src/main/resources/META-INF/resources/render/view.jsp:38]: should we use the `clay:col`, `clay:row` tags instead?

**Comment by jbalsas** [comment]
We're in early stages of dev in `7.4`, so now we should be a bit more aggressive with changes, specially if CI is preventing progress. We've experienced this in the past where we can't get this in soon enough, and by the time we can, it's too late to risk it.

If manual smoke test looks okay and ci is more or less okay too, we should forward. We have a whole year to catch and fix regressions.

@marcoscv-work, could you rebase and push force so we can retry those tests?

File: modules/apps/commerce/commerce-product-type-grouped-web/src/main/resources/META-INF/resources/render/view.jsp (hunk 1 above)
Comment (marcoscv-work) [reply modules/apps/commerce/commerce-product-type-grouped-web/src/main/resources/META-INF/resources/render/view.jsp:38]: **yes**!! :-D
Could we do it after fixing the compatibility task?

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
ci:test:bundle

**Comment by marcoscv-work** [comment]
Files I'm skipping and not adding to the PR:

```modified:   modules/apps/archived/amazon-rankings-web/src/main/resources/META-INF/resources/configuration.jsp
	modified:   modules/apps/archived/directory-web/src/main/resources/META-INF/resources/view_organizations.jsp
	modified:   modules/apps/archived/directory-web/src/main/resources/META-INF/resources/view_user_groups.jsp
	modified:   modules/apps/archived/directory-web/src/main/resources/META-INF/resources/view_users.jsp
	modified:   modules/apps/archived/opensocial-portlet/docroot/adhoc_gadget/configuration.jsp
	modified:   modules/apps/archived/opensocial-portlet/docroot/admin/edit_gadget.jsp
	modified:   modules/apps/archived/opensocial-portlet/docroot/admin/view.jsp
	modified:   modules/apps/archived/opensocial-portlet/docroot/editor/view.jsp
	modified:   modules/apps/archived/powwow-portlet/docroot/admin/edit_server.jsp
	modified:   modules/apps/archived/powwow-portlet/docroot/admin/view.jsp
	modified:   modules/apps/archived/social-activity-web/src/main/resources/META-INF/resources/view.jsp
	modified:   modules/apps/archived/tasks-portlet/docroot/tasks/edit_task.jsp
	modified:   modules/apps/archived/weather-web/src/main/resources/META-INF/resources/configuration.jsp
	modified:   modules/apps/archived/web-form-web/src/main/resources/META-INF/resources/configuration.jsp
	modified:   modules/apps/archived/wysiwyg-web/src/main/resources/META-INF/resources/configuration.jsp
	modified:   modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/FormatTagAttributes.testjsp
	modified:   modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/FormatTagAttributes.testjsp```

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
I'm going to reorder commits and force the last one, then directly to be merged after a @eduardoallegrini or @pat270 review, right?

And then, we can create the task to move to taglibs all grid commerce markup, I think this can be a complex task because I'm seeing a strange grid markup and no needed spacers in commerce

**Comment by jbalsas** [comment]
> And then, we can create the task to move to taglibs all grid commerce markup, I think this can be a complex task because I'm seeing a strange grid markup and no needed spacers in commerce

Sounds good!

**Comment by marcoscv-work** [comment]
Close and continue: https://github.com/liferay-frontend/liferay-portal/pull/499

**Pull request closed without merging**